.github/workflows/patch_updater.yml

```diff
@@ -27,8 +33,24 @@ jobs:
           git config --local user.name "github-actions[bot]"
           git add CMakeLists.txt
           git commit -m "Updated Patch Count"
+
       - name: Push changes # push the output folder to your repo
-        if: "!contains(steps.log.outputs.message, 'Bump version') && contains(steps.repo.outputs.message, 'lnis-uofu/OpenFPGA')"
+        if: "!contains(steps.log.outputs.message, 'Updated Patch Count') && contains(steps.repo.outputs.message, 'lnis-uofu/OpenFPGA')"
         uses: ad-m/github-push-action@master
         with:
           github_token: ${{ secrets.GITHUB_TOKEN }}
+          branch: ${{env.BRANCH_NAME}}
+
+      - name: Create Auto PR
+        if: "!contains(steps.log.outputs.message, 'Updated Patch Count') && contains(steps.repo.outputs.message, 'lnis-uofu/OpenFPGA')"
+        uses: repo-sync/pull-request@v2
+        with:
+          source_branch: ${{env.BRANCH_NAME}}           # name of branch for PR
```

Comment:
Should we remove the source branch after merging? I will update developer guidelines then.